h/cli/commands/search.py

```diff
@@ -39,22 +55,6 @@ def update_settings(ctx):
     _update_settings_old(ctx)
 
 
-def _reindex_old(ctx):
```

Comment:
I moved the content of this function into the `reindex` function above since the same code works for both ES versions for the most part.